docs/features/feature-guides/properties/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Overview
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub Structured Properties
+<FeatureAvailability/>
+
+**DataHub Structured Properties** provide a structured and validated approach to attaching metadata to your Data Assets, enabling organizations to enforce consistency and quality in metadata collection.
+
+<p align="center">
+  <img width="90%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/main/imgs/properties/custom_and_structured_properties.png"/>
+</p>
+
+## What are Structured Properties?
+
+**Structured Properties** are a configurable feature in DataHub that allow metadata to be associated with Assets in a structured format. Each Structured Property has a specific type (e.g., Date, Integer, DataHub Asset, Text) and can include validation rules, such as restricting to specific allowed values or adhering to a pre-defined format.
+
+Structured Properties can be added to all DataHub Asset types, including Datasets, Dashboards, Pipelines, and more. This ensures flexibility in managing metadata across your entire data ecosystem.
```

Comment:
I'd just say:

Structured Properties can be applied to the following entity types:

- X
- Y
- Z
... comprehensive list. 


This is more useful I think and we can drop the second sentence

Reply:
How about this:

> Structured Properties can be added to the following Asset Types:
> 
> - Data Assets, such as Datasets, Columns, Tasks, Pipelines, Charts, Dashboards, and more.
> - DataHub Entities, such as Domains, Glossary Terms & Groups, and Data Products.

Fully listing out _all_ Asset types will be a very long list